webapp/src/features/adminAddNotification/AdminAddNotification.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect } from 'react'
+import { Form } from 'react-bootstrap'
+import { useForm } from 'react-hook-form'
+import { DropdownList } from 'react-widgets'
+import {
+  selectSuccessMsg,
+  selectSelectedType,
+  selectAvailableTypes,
+  selectApiData,
+  selectErrorState,
+  selectErrorMsg,
+  selectSubmitAttempt,
+
+  // actions
+  submitForm,
+  AdminNotificationForm,
+  setSelectedType,
+  updateEmailTypesApiData,
+  getNotificationData,
+} from './adminAddNotificationSlice'
+import { selectEmail } from '../../generalSlices/userSlice'
+import { useSelector, useDispatch } from 'react-redux'
+
+import '../adminRsuTab/Admin.css'
+import 'react-widgets/styles.css'
+import { AnyAction, ThunkDispatch } from '@reduxjs/toolkit'
+import { RootState } from '../../store'
+
+const AdminAddNotification = () => {
+  const dispatch: ThunkDispatch<RootState, void, AnyAction> = useDispatch()
+  const successMsg = useSelector(selectSuccessMsg)
+  const apiData = useSelector(selectApiData)
+  const errorState = useSelector(selectErrorState)
+  const errorMsg = useSelector(selectErrorMsg)
+  const submitAttempt = useSelector(selectSubmitAttempt)
+  const selectedType = useSelector(selectSelectedType)
+  const availableTypes = useSelector(selectAvailableTypes)
+  const userEmail = useSelector(selectEmail)
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm<AdminNotificationForm>()
+
+  useEffect(() => {
+    dispatch(getNotificationData())
+  }, [dispatch])
+
+  useEffect(() => {
+    dispatch(updateEmailTypesApiData())
+  }, [apiData, dispatch])
+
+  const onSubmit = (data: AdminNotificationForm) => dispatch(submitForm({ data, reset }))
+
+  return (
+    <div>
+      <Form onSubmit={handleSubmit(onSubmit)}>
+        <Form.Group className="mb-3" controlId="email">
+          <Form.Label>Email</Form.Label>
+          <Form.Control type="email" defaultValue={userEmail} value={userEmail} {...register('email')} readOnly />
```

Comment:
I am not sure how intuitive the email being an un-editable textbox is. I feel like it makes it seem like it can be edited. I understand it is probably easier to grab the values from the form this way but having it not be in a textbox would probably be the least deceptive way of displaying the email.

Reply:
Updated to display just the email instead of a disabled input.